### DISCUSSION TITLE
essay/info/HJ9NrwElzyArPbCm-Exeh

### DISCUSSION THREAD

**Discussion by giscus** [wurencaideli/dumogu-ypwz-comment#19] [Announcements]
# essay/info/HJ9NrwElzyArPbCm-Exeh

不是只有视频和音乐才带给人欢乐！还有一篇文字哟！学生时代所看过的短文还有谁记得呢？甚是怀念呢！

https://www.dumogu.top/essay/info/HJ9NrwElzyArPbCm-Exeh?isInfo=false

**Comment by wurencaideli** [comment]
光脚的不怕穿鞋的，有些人的思想是真的有问题
